rclpy/rclpy/node.py

```diff
@@ -146,8 +146,8 @@ def __init__(
         self.__guards: List[GuardCondition] = []
         self.__waitables: List[Waitable] = []
         self._default_callback_group = MutuallyExclusiveCallbackGroup()
+        self._parameters_callback: List[Callable[[List[Parameter]], SetParametersResult]] = []
```

Comment:
Maybe this should be renamed to `self._parameters_callback_list`

Reply:
Just making `callback` plural should be fine. Same as `guards` / `waitables` above.